src/rpc/blockchain.cpp

```diff
@@ -1331,6 +1331,7 @@ static RPCHelpMan gettxout()
                 {RPCResult::Type::STR_AMOUNT, "value", "The transaction value in " + CURRENCY_UNIT},
                 {RPCResult::Type::OBJ, "scriptPubKey", "", {
                     {RPCResult::Type::STR, "asm", ""},
+                                {RPCResult::Type::STR, "desc", "Inferred descriptor for the output\n"},
```

Comment:
In c90a08886afe86d84bbd1d76ae3d3ecc24f24a74 "transaction decoding infer output descriptors"

Indentation, also an extraneous `\n`